### DISCUSSION TITLE
Add uptime-latest.json and response-time-latest.json.

### DISCUSSION THREAD

**Discussion by FxllenCode** [upptime/upptime#201] [Ideas]
This is stemming from #194.

<hr>

## Feature Request: We need an endpoint that returns the current status!

I opened an issue yesterday as I assumed that uptime.json and response-time.json return the current information. I found that these do not actually return that; they return all time information. After doing even more searching, there isn't an endpoint that returns the current status.

I propose that there is an uptime-latest.json and response-time-latest.json. This would greatly improve this system, as I'm trying to use this into a live game and send an API request to get the current status, however there isn't an endpoint that provides this.

Thanks! 🙂

**Comment by FxllenCode** [comment]
I could really use this now... the only way of getting the current status is through the YAML, which cannot be parsed but the application I'm using. It makes the API portion of upptime obsolete if we cannot get the current status.

**Reply by AnandChowdhary** [reply]
I'll make sure I prioritize this feature!

**Reply by FxllenCode** [reply]
Will the latest releases include this? I'm wondering how this might work as I might end up having to work on a YAML - Lua parser if this feature isn't possible.

**Reply by FxllenCode** [reply]
Hi @AnandChowdhary, any update? For now, I am just using the basic summary.json but I think the API would be really simplified if each endpoint had it's own response-latest and uptime-latest.

Thanks!

**Reply by FxllenCode** [reply]
Hey @AnandChowdhary just asking again, will this feature see production? Would be super useful, maybe others can submit a PR as well? I am not sure how to go about making this.

**Reply by XXCoreRangerX** [reply]
I agree, this would be a useful feature. I second this!

**Reply by AnandChowdhary** [reply]
Absolutely! We're already working on prioritizing top requested features and this is one of them!
